kaggle/README.md

```diff
@@ -0,0 +1,28 @@
+# Kaggle Community Connector for Data Studio
+![Screenshot](./KaggleSampleDataset.png?raw=true "Screenshot")
+*This is not an official Google product*
+
+This [Data Studio](https://datastudio.google.com) [Community
+Connector](https://developers.google.com/datastudio/connector) lets users import any dataset from
+[kaggle](https://www.kaggle.com). This Community
+Connector uses the [Kaggle API](https://github.com/Kaggle/kaggle-api).
+
+## Try the Community Connector in Data Studio
+
+You can try out the managed deployment of the latest code: [Kaggle Community Connector](https://datastudio.google.com/datasources/create?connectorId=AKfycbzRfJ2ofuhRGSEc)
```

Comment:
Please exclude the managed deployment. We'll manage that after the code is merged in.